### DISCUSSION TITLE
Warnstufen

### DISCUSSION THREAD

**Discussion by threadstone83** [stephan192/hochwasserportal#39] [Ideas]
Hallo @stephan192 

ist es evtl. möglich auch die einzelnen Warnstufen mit auszugeben? Ich weiß nicht wie es in den anderen Bundesländern aussieht, aber in Bayern werden sie wohl mit angegeben:

https://www.hnd.bayern.de/pegel/unterer_main/bad-kissingen-prinzreg-bau-24405002

Viele Grüße
Andreas

**Comment by stephan192** [comment]
Hallo @threadstone83,
ich bin mir nicht ganz sicher welche Werte mit "die einzelnen Warnstufen" gemeint sind. Sind es im konkreten Fall die Meldestufen 1-4 sprich die Werte 250 cm, 300 cm, 350 cm und 400 cm?

Falls diese Werte gemeint sind macht es aus meiner Sicht keinen Sinn diese als z.B. weitere Attribute auszugeben/anzuzeigen. Bei den Werten handelt es sich um Konstanten die sich so gut wie nie ändern und die für sich alleine nicht von Relevanz sind. Relevant sind die Meldestufen die ausgegeben/angezeigt werden. Würde man die cm-Werte ausgeben erhält man keinen Mehrwert sondern nur redundante Informationen.
Zusätzlich kommt hinzu, dass z.B. für die bayerischen Pegel die exakten Werte nicht ausgelesen, gespeichert und verarbeitet werden müssen um auf die finale Information (die Meldestufe) zu kommen.

**Reply by threadstone83** [reply]
Ja, diese Werte hatte ich gemeint. Aber ich gebe dir Recht. Da es fixe Werte sind, kann man sich diese auch heraussuchen und dann verarbeiten (ich will die Meldestufen im Diagramm mit darstellen).
